src/sia.js

```diff
@@ -27,6 +27,9 @@ const call = (address, opts) => new Promise((resolve, reject) => {
 	}
 	callOptions.url = 'http://' + address + callOptions.url
 	callOptions.json = true
+	if (typeof callOptions.timeout === 'undefined') {
+		callOptions.timeout = 10000
+	}
```

Comment:
this is just slightly-safer longhand for `callOptions.timeout = callOptions.timeout || 10000`, right?